scripts/zones/East_Sarutabaruta/IDs.lua

```diff
@@ -25,7 +25,6 @@ zones[xi.zone.EAST_SARUTABARUTA] =
         SIGNPOST_OFFSET          = 7382, -- Southeast: South Tower, Horutoto Ruins Southwest: Windurst Woods
         TABY_CANATAHEY_DIALOG    = 7392, -- This is the entrrrance to Windurst. Please maintain orderrrly conduct while you'rrre in town.
         HEIH_PORHIAAP_DIALOG     = 7393, -- These grrrasslands make up East Sarutabaruta. Lately the number of monsters has drrramatically increased, causing us all sorts of trrrouble.
-        SAMA_GOHJIMA_PREDIALOG   = 7395, -- The ministerrr of the Orastery is in the laborrratory beneath here. To get there, you should check the walls verrry carrrefully.
```

Comment:
Why remove this?

Reply:
The DefaultAction given to her is the same dialogue, only in event form.